assets/js/components/KeyMetrics/MetricsSelectionPanel/SelectionPanelFooter.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Selection Panel Footer component.
+ *
```

Comment:
Included key metrics specific component, in order to override the inline error, since global one is used in Audience segmentation. The new error component added in this PR can be reused there as well for consistency, but is outside of scope of this PR. Will ping team M about this